docs/UserGuide.md

```diff
@@ -333,18 +333,20 @@ Format: `guest filter [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [r/RSVP_STATUS] [
 ><br>
 >- Parameters in square brackets are optional. However, you must include at least one parameter to filter by.
    ><br>
->- NAME cannot be filtered with empty values.
-   ><br>
->- For all parameters aside from NAME, providing an empty value will filter for guests with unfilled values for that parameter.
+>- For all parameters aside from NAME and RSVP_STATUS, providing an empty value will filter for guests with unfilled values for that parameter.
    ><br>
 >- Providing an empty `r/` value would filter for guests with unknown RSVP status.
    ><br>
+>- Providing an empty `n/` value would filter for guests without a name thus returning an empty guest list. (All guests need a name)
+   ><br>
```

Comment:
Ideally an empty `n/` should throw an error because there is no point filtering for an empty name. If the implementation for that is difficult, perhaps we can stick with this!

Reply:
Resolved!